api/payara-bom/pom.xml

```diff
@@ -341,7 +341,7 @@
             <dependency>
                 <groupId>com.fasterxml.woodstox</groupId>
                 <artifactId>woodstox-core</artifactId>
-                <version>${woodstox.version}</version>
+                <version>${woodstox-core.version}</version>
```

Comment:
By my suggested "use project names" convention this should probably remain as `woodstox.version`